src/SetupModel.cpp

```diff
@@ -184,7 +191,12 @@ void SetupModel(char* DensityFile, char* NetworkFile, char* SchoolFile, char* Re
 		P.LocationInitialInfection[i][0] -= P.SpatialBoundingBox[0];
 		P.LocationInitialInfection[i][1] -= P.SpatialBoundingBox[1];
 	}
+	// Find longest distance - may not be diagonally across the bounding box.
 	t = dist2_raw(0, 0, P.width, P.height);
+	double tw = dist2_raw(0, 0, P.width, 0);
```

Comment:
I'm not sure exactly what calculation is being done here, but does it need to check `dist2_raw(0, P.height, P.width, P.height);` too, for the other hemisphere? And does it need to do something different for regions that span the equator?

Reply:
The `dist2_raw` is calculating the square of the great circle distance between two points (see src/Dist.cpp dist2UTM - [this](https://www.movable-type.co.uk/scripts/latlong.html) is a useful website that gives the formula and what's going on.

What the model does is read in the geography and then rotate it so that the bottom left of the bounding box is at (0 latitude, 0 longitude).  
 This is fine because the Earth is modelled as a perfect sphere.  Therefore, we don't have to worry about countries that straddle the equator, or about `dist2_raw(0, P.height, P.width, P.height)` as that will be less than `dist2_raw(0, 0, P.width, 0).  What we do possibly have to do is alert on countries that are more than 180 degrees wide or 90 degrees tall.